src/rpcblockchain.cpp

```diff
@@ -1063,1371 +1097,2344 @@ int64_t AmountFromDouble(double dAmount)
+    std::string Narr = ToString(GetAdjustedTime());
+
+    c.push_back(Pair("Lifetime Payments Report", Narr));
+    results.push_back(c);
+
+    LOCK(cs_main);
+
+    CBlockIndex* pindex = pindexGenesisBlock;
+
+    while (pindex->nHeight < pindexBest->nHeight)
     {
-        TallyMagnitudesInSuperblock();
-        entry.push_back(Pair("Done","Done"));
-        results.push_back(entry);
+        pindex = pindex->pnext;
+
+        if (pindex==NULL || !pindex->IsInMainChain())
+            continue;
+
+        if (pindex == pindexBest)
+            break;
 
+        if (pindex->GetCPID() == cpid && (pindex->nResearchSubsidy > 0))
+            res.push_back(Pair(ToString(pindex->nHeight), RoundToString(pindex->nResearchSubsidy, 2)));
     }
-    else if (sItem == "addkey")
+    //8-14-2015
+    StructCPID stCPID = GetInitializedStructCPID2(cpid, mvResearchAge);
+
+    res.push_back(Pair("Average Magnitude", stCPID.ResearchAverageMagnitude));
+    results.push_back(res);
+
+    return results;
+}
+
+Value magnitude(const json_spirit::Array& params, bool fHelp)
+{
+    if (fHelp || params.size() > 1)
+        throw runtime_error(
+                "magnitude <cpid>\n"
+                "\n"
+                "<cpid> -> Required to be used on mainnet\n"
+                "\n"
+                "Displays information for the magnitude of all cpids or specified in the network\n");
+
+    Array results;
+
+    std::string cpid = "";
+
+    if (params.size() > 1)
+        cpid = params[0].get_str();
+
     {
-        //To whitelist a project:
-        //execute addkey add project projectname 1
-        //To blacklist a project:
-        //execute addkey delete project projectname 1
-        //Key examples:
-
-        //execute addkey add project milky 1
-        //execute addkey delete project milky 1
-        //execute addkey add project milky 2
-        //execute addkey add price grc .0000046
-        //execute addkey add price grc .0000045
-        //execute addkey delete price grc .0000045
-        //GRC will only memorize the *last* value it finds for a key in the highest block
-
-        //execute memorizekeys
-        //execute listdata project
-        //execute listdata price
-
-        if (params.size() != 5)
-        {
-            entry.push_back(Pair("Error","You must specify an action, message type, message name and value."));
-            entry.push_back(Pair("Example","execute addkey add_or_delete keytype projectname value."));
-            results.push_back(entry);
-        }
-        else
-        {
-            //execute addkey add project grid20
-            std::string sAction = params[1].get_str();
-            entry.push_back(Pair("Action",sAction));
-            bool bAdd = (sAction=="add") ? true : false;
-            std::string sType = params[2].get_str();
-            entry.push_back(Pair("Type",sType));
-            std::string sPass = "";
-            sPass = (sType=="project" || sType=="projectmapping") ? GetArgu
```

Comment:
These comments should probably have "execute" removed from all of them (lines 2094-2110)